tensorflow/tensorflow-0.10.0rc0.patch

```diff
@@ -0,0 +1,25 @@
+diff -ruN tensorflow-master/tensorflow/cc/BUILD tensorflow-master-patch/tensorflow/cc/BUILD
+--- tensorflow-master/tensorflow/cc/BUILD	2016-02-06 10:08:02.000000000 +0900
++++ tensorflow-master-patch/tensorflow/cc/BUILD	2016-02-06 17:42:56.446230724 +0900
+@@ -75,6 +75,21 @@
+     ],
+ )
+
++cc_binary(
++    name = "libtensorflow.so",
```

Comment:
They've added that target only recently. If it does what we need, let's use that, sure. In any case, remove the patch for v0.9.0. Thanks!